sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/api/v2/schemas/namespaces/model/dto/NamespaceDto.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2.docsapi.api.v2.schemas.namespaces.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import java.util.List;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+/** DTO for namespace datacenter. */
+public record NamespaceDto(
+
+    // TODO should we return replicas count in case of the simple strategy
```

Comment:
I think we could return number of replicas - since it's just additional data and would be non-breaking, nothing wrong with that

Reply:
Hmm, I am not sure about this anymore.. Seems that replicas count in the `SimpleStrategy` is not really taking any effect, I set `3` when creating, but then get it back as `1`.. And the query is fine and everything.. We should open a separate ticket for this, it might be that is actually depending on the nodes in the cluster? Or?